molgenis-data-postgresql/src/main/java/org/molgenis/data/postgresql/PostgreSqlExceptionTranslator.java

```diff
@@ -85,6 +85,7 @@ private MolgenisDataException doTranslate(SQLException sqlException) {
       sqlException = sqlException.getNextException();
     }
     if (!(sqlException instanceof PSQLException)) {
+      LOG.error("", sqlException);
```

Comment:
The contract with the framework is to return null and let some fallback exception handler kick in.